docs/libs/fs.rst

```diff
@@ -15,6 +15,7 @@ LVG has built in support for:
 - WIN32 (Windows using Win32 API function .e.g ``CreateFileA``, ``ReadFile``)
 - MEMFS (read a file from a memory buffer)
 - LITTLEFS (a little fail-safe filesystem designed for microcontrollers)
+- Arduino LITTLEFS (a little fail-safe filesystem designed for Arduino ESP)
```

Comment:
If this Arduino LittleFS is really ESP dependent, then name should reflect that also!
```suggestion
- Arduino ESP LITTLEFS (a little fail-safe filesystem designed for Arduino ESP)
```